source/jobs/README.md

```diff
@@ -0,0 +1,232 @@
+
+  * [Jobs Feature](#jobs-feature)
+    + [Creating a Job](#creating-a-job)
+    + [Creating your own Job Handler](#creating-your-own-job-handler)
+      - [Job/Job Handler Security Considerations](#jobjob-handler-security-considerations)
+    + [Debugging your Job](#debugging-your-job)
+    + [Jobs Build Flags](#jobs-build-flags)
+    + [Jobs Feature Configuration Options](#jobs-feature-configuration-options)
+      - [Configuring the Jobs feature via the command line](#configuring-the-jobs-feature-via-the-command-line)
+      - [Configuring the Jobs feature via the JSON configuration file](#configuring-the-jobs-feature-via-the-json-configuration-file)
+
+[*Back To The Main Readme*](../../README.md)
+
+## Jobs Feature
+
+The Jobs feature within the AWS IoT Device Client provides device-side functionality for execution of jobs created via
+https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html. When the Jobs feature starts within the AWS IoT Device Client, 
+the feature will attempt to establish subscriptions to important notification topics, and will also publish
+a request to receive the latest pending jobs. It will use the shared MQTT connection to subscribe/publish to these
+topics. 
+
+When a new Job is received via these topics, the Jobs feature will look for and extract the following required and optional elements:
+
+
+### Creating a Job
+
+[View a sample job document here](sample-job-docs/install-packages.json)
+
+The following fields are extracted from the incoming Job document by the Jobs feature within the AWS IoT Device Client when a 
+job is received.
+
+`operation` *string* (Required): This is the executable or script that you'd like to run. Here we can specify a script located
+in the Device Client's handler directory, such as `install-packages.sh`, or it could be a well known executable binary
+in the Device Client's environment path, such as `echo` or `apt-get`.
+
+`args` *string array* (Optional): This is a JSON array of string arguments which will be passed to the specified operation. For example, 
+if we want to run our `install-packages.sh` to install the package `ifupdown`, our args field would look like:
+```
+...
+"args": ["ifupdown"],
+...
+```
+If we wanted our remote device to print out "Hello World", then our job document so far might look like:
+```
+{
+    "operation": "echo",
+    "args": ["Hello world!"]
+}
+```
+
+`path` *string* (Optional): This field tells the Jobs feature where it should look to find an executable that matches the specified
+operation. If this field is omitted, the Jobs feature will assume that the executable can be found in its path. If
```

Comment:
Do you mean the PATH environment variable? or some custom path in the device client?

Reply:
this is a variable in the job docs file(s)

Reply:
Sorry, should've clarified I was referring to the `path` in the sentence, `If this field is omitted, the Jobs feature will assume that the executable can be found in its path ` not the `path` in `path *string* (Optional): This field tells the Jobs feature where it should`

Reply:
Oh yes ill update that, in the sentence that is the environment variable PATH